akka-actor-tests/src/test/scala/akka/actor/dispatch/DispatcherActorSpec.scala

```diff
@@ -10,6 +10,7 @@ import java.util.concurrent.{ CountDownLatch, TimeUnit }
 import java.util.concurrent.atomic.AtomicBoolean
 import akka.testkit.AkkaSpec
 import akka.actor.{ Actor, Props }
+import akka.actor.actorRef2Scala
```

Comment:
https://dotty.epfl.ch/docs/reference/dropped-features/package-objects.html

Reply:
Hmm. The link is just for reference purpose or you think there is a better way than adding  `import akka.actor.actorRef2Scala`?